gcloud/bigquery/dataset.py

```diff
@@ -13,13 +13,10 @@
 # limitations under the License.
 
 """Define API Datasets."""
-
-import datetime
-
-import pytz
 import six
 
 from gcloud.exceptions import NotFound
+from gcloud.bigquery._helpers import _datetime_from_prop
```

Comment:
I'm doing what I "like" (alphabetical, with sections for stdlib, 3rd party and local) and you are doing what you "like". Maybe we should codify and not leave it up to our whims. (Though way more pressing things at this time.)